packages/extension-api/src/extension-api.d.ts

```diff
@@ -3715,6 +3715,16 @@ declare module '@podman-desktop/api' {
      * The system clipboard.
      */
     export const clipboard: Clipboard;
+
+    /**
+     * Indicates if the user is using WSL to run podman machines
+     */
+    export function isWSL(): Promise<boolean>;
+
+    /**
+     * Indicates if the user is using HyperV to run podman machines
+     */
+    export function isHyperV(): Promise<boolean>;
```

Comment:
this can't be provided like this in env

env is generic, you can have like no podman or no podman machine running at all

Reply:
So should this be just part of the podman extension?